test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.2
```

Comment:
https://github.com/kubernetes/kubernetes/pull/78389/files was updated to 2.0 instead of 1.1, so this should be 2.1, correct?

Reply:
Yep.